styleguide/modules/ROOT/pages/placeholders.adoc

```diff
@@ -1,9 +1,32 @@
 = Code Placeholders
 
-When you need to add a placeholder value to a xref:examples.adoc[code example]: 
+The type of placeholder you add to code depends on the type of xref:examples.adoc[code example] you're including. 
+
+== curl and Shell Placeholders
+
+When you need to add a placeholder value to a curl or other shell code example: 
 
 * Write the placeholder name in capital letters. 
 * Use a `$` before the placeholder name. 
-* Use underscores to separate words. 
+* Use underscores (`_`) to separate words. 
+
+For example, `$DATABASE_NAME`. 
+
+== REST API Path Attribute Placeholders 
+
+When you need to add a placeholder value to a path attribute for a REST API example: 
+
+* Write the placeholder name in capital letters. 
+* Surround the placeholder name in curly braces (`{}`).
+* If necessary, use underscores (`_`) to separate words.
+
+For example, `{PORT}`.
+
+== Other Placeholders 
```

Comment:
Is it worth emphasizing that this is the default formatting for placeholders?

Reply:
I don't know about "default" - I think I'd rather just leave it as "other" - since we want the other two options to be used where appropriate, first.